ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -351,11 +375,20 @@ export const SimulationDetails: React.FC<SimulationDetailsProps> = ({
     >
       <Box display={Display.Flex} flexDirection={FlexDirection.Column} gap={3}>
         {staticRows.map((staticRow, index) => (
-          <BalanceChangeList
-            key={index}
-            heading={staticRow.label}
-            balanceChanges={staticRow.balanceChanges}
-          />
+          <>
+            <BalanceChangeList
+              key={index}
+              heading={staticRow.label}
+              balanceChanges={staticRow.balanceChanges}
+              labelColor={getAlertTextColors(selectedAlertSeverity)}
+            />
+            <Box marginLeft={1}>
+              <InlineAlert
```

Comment:
Minor, ideally we'd be able to wrap this in a standard `ConfirmInfoAlertRow` to avoid the duplicated logic but I appreciate that might need more time so not urgent.